doajtest/unit/test_formcontext_emails.py

```diff
@@ -464,15 +464,15 @@ def test_02_ed_review_emails(self):
                                         re.DOTALL)
         assert bool(assEd_email_matched)
 
-        publisher_template = 'publisher_application_editor_assigned.txt'
+        publisher_template = 'publisher_update_request_editor_assigned.txt'
```

Comment:
Same as above, looks like this test no longer checks to see if `publisher_application_editor_assigned` is triggered.